### PR TITLE
jest test for api/items/:id enpoint

### PR DESCRIPTION
**Pull request by marcusbarnesdeveloper**

### CODE REVIEW

**Comment by marcusbarnesdeveloper** [comment]
@geophray please review my PR

File: server/test/server.test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+   done();
+
+ });
+
+ test("It should  contain correct array items", async (done) => {
+  const mainImage = [
+    "https://i.picsum.photos/id/492/200/200.jpg",
+    "https://i.picsum.photos/id/493/200/200.jpg",
+    "https://i.picsum.photos/id/494/200/200.jpg",
+    "https://i.picsum.photos/id/495/200/200.jpg"
```
Comment (geophray) [inline server/test/server.test.js:42]: These images will eventually be uploaded to and served from S3, correct?

Comment (marcusbarnesdeveloper) [reply server/test/server.test.js:42]: yeah that is the plan. But right now those are just some random photos.

**Comment by marcusbarnesdeveloper** [comment]
@geophray added a newline to end of file

**Pull request merged**